tools/schemacode/bidsschematools/data/metaschema.json

```diff
@@ -0,0 +1 @@
+../../../../src/metaschema.json
```

Comment:
This gets resolved by setuptools during `sdist` build, so it's safe to use a symlink here.